README.md

```diff
@@ -161,12 +161,16 @@ The decorator processes all immediately available commands in the commands-chan,
                      :group-id                       "example"
                      :value-type                     :string
                      :shape                          :value
-                     :ketu.source/consumer-decorator (fn [consumer-ctx poll-fn]
-                                                       (loop []
-                                                         (when-let [command (async/poll! commands-chan)]
-                                                           (command consumer-ctx)
-                                                           (recur)))
-                                                       (poll-fn))}
+                     :ketu.source/consumer-decorator (reify ConsumerDecorator
```

Comment:
suggestion: add a helper function `->ConsumerDecorator` to make the caller code cleaner